ui/scripts/cloudStack.js

```diff
@@ -329,7 +325,6 @@
                         g_regionsecondaryenabled = null;
                         g_loginCmdText = null;
                         
-                        $.cookie('JSESSIONID', null);
                         $.cookie('sessionkey', null);
```

Comment:
I'm no UI/Javascript expert, but should these include hostname as well? If so, same goes for L351+